README.md

```diff
@@ -90,6 +90,9 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [Gitignore.io](https://www.gitignore.io) Create Useful .gitignore Files For Your Projects
 * [Exercism](http://exercism.io/) Code practice and mentorship for everyone
 
+### Android developpement
```

Comment:
Make it similar to the two previous titles (Learn data science and Learn to code) and update the summary (line 8-9)

```suggestion
### Learn Android development
```